client/lib/wpcom-xhr-wrapper/index.js

```diff
@@ -20,7 +20,7 @@ export default async function ( params, callback ) {
 	return xhr( params, function ( error, response, headers ) {
 		if ( error && error.name === 'InvalidTokenError' ) {
 			debug( 'Invalid token error detected, authorisation probably revoked - logging out' );
-			userUtils.logout();
+			redirectToLogout();
```

Comment:
This doesn't have access to `user.logout_URL` and therefore will always use the `config( 'logout_url' )` fallback without a nonce.

Reply:
That's a fair point. Do you have any recommendations with regards to passing the user here? Using `redux-bridge` perhaps?